src/sagemaker_xgboost_container/algorithm_mode/train.py

```diff
@@ -283,7 +293,9 @@ def train_job(train_cfg, train_dmatrix, val_dmatrix, train_val_dmatrix, model_di
                 logging.debug("Stored trained model {} at {}".format(fold, model_location))
 
 
-def get_callbacks_watchlist(train_cfg, train_dmatrix, val_dmatrix, model_dir, checkpoint_dir, is_master, fold=None):
+def get_callbacks_watchlist(train_dmatrix, val_dmatrix, model_dir, checkpoint_dir,
```

Comment:
This arg list is getting pretty long, do you think we can turn this sig into named args?

Reply:
done